docs/oauth-2.0.md

```diff
@@ -345,20 +346,17 @@ end-user's data, Service Accounts provide access to the client application's
 own data. Your client application signs the request for an access token using
 a private key downloaded from the [Google API Console][console].
 
-Example code taken from [plus-serviceaccount-cmdline-sample][plus-sample]:
```

Comment:
Are we able to remove the corresponding URL from the link references at the bottom of this file?